### DISCUSSION TITLE
Error in Docker Compose host_mnt does not exist

### DISCUSSION THREAD

**Discussion by psiu** [Budibase/budibase#3149] [Help] [answered]
Received the following error upon initial run of "docker-compose --env-file hosting.properties up"

Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/host_mnt/Users/_______/Budibase/envoy.yaml" to rootfs at "/etc/envoy/envoy.yaml" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

The "/host_mnt/" folder doesn't exist and there doesn't appear to be an configuration option in the files to adjust otherwise.

On Mac BigSur 11.6.  Docker Desktop 4.1.1

**Comment by shogunpurple** [comment, marked as answer]
Hi @psiu,

It looks like you do not have your `envoy.yaml` in the same directory as your `docker-compose.yaml` file. This file needs to exist so that it can be used by the envoy container.

To make sure that your `envoy.yaml` is in the correct directory, you can pull it with the following command:
`wget https://raw.githubusercontent.com/Budibase/budibase/master/hosting/envoy.yaml`

Could you paste the output of an `ls` in the directory where you are running `docker-compose --env-file hosting.properties up`

**Reply by psiu** [reply]
For whatever reason, the default install created envoy.yaml as a directory and the actual yaml file as envoy.yml
I deleted envoy.yaml and renamed envoy.yml to envoy.yaml - and all is well, thanks
